variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "prefix" {
```

Comment:
Looks like this is unused so you can just remove it. The intention was that you include this in the `name` of each resource. So you could easily use the same terraform config to spin up multiple copies of the app with different resource names.